app/controllers/v1/benchmarks_controller.rb

```diff
@@ -30,5 +28,9 @@ def scope
     def resource
       Xccdf::Benchmark
     end
+
+    def serializer
+      BenchmarkSerializer
+    end
```

Comment:
Changes in this file are to allow users to see which rules exist on a benchmark, via its relationships and optional includes, so they can select rules for tailoring a profile.